README.md

```diff
@@ -324,6 +325,25 @@ params.MinSplitSize = 7168
 totalDownloaded, totalFailed, err := rtManager.DownloadFiles(params)
 ```
 
+#### Downloading Release Bundles from Artifactory
+Using the `DownloadFiles()` function, we can download release bundles and get the general statistics of the action (The actual number of files downloaded, and the number of files we expected to download), and the error value if it occurred.
+
+It is possible to validate the downloaded release bundle's files by providing a local path to a GPG public key file (which used to sign the release bundle). 
+
+
+
+
+```go
+params := services.NewDownloadParams()
+params.Target = "target/path/"
+// Bundle's name and version should be separated with "/".
+params.Bundle = "bundleName/10"
+// Optional GPG validation
+params.PublicGpgKey = "public/key/file/path"
+totalDownloaded, totalFailed, err := rtManager.DownloadFiles(params)
+```
+Read more about uploading and signing with [GPG key](https://www.jfrog.com/confluence/display/JFROG/GPG+Signing).
```

Comment:
Read more about uploading and signing with [GPG key](https://www.jfrog.com/confluence/display/JFROG/GPG+Signing).
-->
Read more about GPG signing release bundles [here](https://www.jfrog.com/confluence/display/JFROG/GPG+Signing).